server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -259,6 +269,69 @@ private void blockUntilShutdown() throws InterruptedException {
     server.blockUntilShutdown();
   }
 
+  public ServerStatus getServerStatus() {
+    return serverStatus;
+  }
+
+  public void decommission() {
+    checkStatusForDecommission();
+    synchronized (statusLock) {
+      checkStatusForDecommission();
+      long checkInterval = shuffleServerConf.get(SERVER_DECOMMISSION_CHECK_INTERVAL);
+      decommissionThread = new Thread(() -> {
+        while (isDecommissioning()) {
```

Comment:
When server is cancelled decommission, it's better to log this.

Reply:
Updated